plugins/removeComments.js

```diff
@@ -5,6 +5,12 @@ const { detachNodeFromParent } = require('../lib/xast.js');
 exports.name = 'removeComments';
 exports.description = 'removes comments';
 
+/**
+ * If a comment matches one of the following patterns, it will be
+ * preserved by default. Particularly for copyright/license information.
+ */
+const DEFAULT_PRESERVE_PATTERNS = [/^!/, /LICENSE/i, /COPYRIGHT/i];
```

Comment:
Does anyone here know the motivation for the `!` check?
With the new preserve patterns param, perhaps that one can be removed in v4?